pyproject.toml

```diff
@@ -94,7 +94,7 @@ pyupgrade = "^2.11.0"
 Sphinx = "4.1.1"
 
 [tool.poetry.extras]
-prediction = ["tensorflow", "pmdarima", "transformers", "torch"]
+prediction = ["tensorflow", "pmdarima"]
```

Comment:
Are we removing transformers and torch?

Reply:
These have been long gone (whenever I added vader for twitter)